integration-tests/drivers/TestDriver063/android/app/build.gradle

```diff
@@ -194,7 +193,6 @@ dependencies {
     implementation "com.facebook.react:react-native:+"  // From node_modules
 
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
-    implementation "com.heapanalytics.android:heap-android-client:$heapVersion"
```

Comment:
Careful. I think we still want `heap-android-client` (the actual library, as opposed to the plugin).

Reply:
Ah sorry, I think I missed what you did in `./android/build.gradle`. In that case, I think you might be okay here actually.